src/redux/configstore.js

```diff
@@ -0,0 +1,17 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import covidReducer from './covid';
+import detailsReducer from './details';
+
+export const reducer = combineReducers({
+  covidReducer,
+  detailsReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(thunk, logger),
+);
+
```

Comment:
- [ ] Please do not log redux data on the browser console as it may cause site security issues. Kindly clear the logged data and errors displayed on the browser console.